src/style.css

```
@@ -1,8 +1,29 @@
 body {
-  margin: 0;
-  padding: 0;
+  padding: 0; 
+  margin:0;
 }
 
+footer {
+  text-align: center;
+  font-family: Outfit, sans-serif;
+  position: relative;
+  width: 100%; 
+  font-style: normal;
+  height: 100px;
+  display: flex; 
+  align-items: center; 
+  justify-content: center;
+  background-color: #BDE3D0;
+  gap: 20px;
+}
+
+  footer img {
+    width: 30px;
+    height: 30px;
+    }
```

There seems to be unnecessary spacing for this. Remind me to see if prettier is working for everyone :)

```suggestion
footer img {
  width: 30px;
  height: 30px;
}
```